client/components/App.jsx

```diff
@@ -9,9 +9,9 @@ const App = () => {
   return (
     <React.Fragment>
       <h1>React development has begun!</h1>
-      <Route SignIn path="/signin" component={SignIn} />
-      <Route Dashboard path="/" component={Dashboard} />
-      <Route path="/register" component={RegistrationForm} />
+      <Route exact path="/signin" component={SignIn} />
```

Comment:
this should be using a <Switch> statement from react-router-dom wrapped around all routes, not exact path.
Also is this part of your ticket?